frontend/src/metabase-types/api/query.ts

```diff
@@ -7,6 +7,7 @@ import type { TableId } from "./table";
 export interface StructuredQuery {
   "source-table"?: TableId;
   aggregation?: AggregationClause;
+  limit?: number;
```

Comment:
Added based on `/api/card/{cardId}`, in which `dataset_query > query` looks like:
```
{
    "source-table": 2,
    "limit": 1,
}
```